Intersect.Server/Entities/Events/Conditions.cs

```diff
@@ -614,6 +614,11 @@ Event instance
                     compValue = player.User.GetVariableValue(comparison.CompareVariableId);
                 }
             }
+            else if(comparison.TimeSystem == true)
+            {
+                compValue = (long)(DateTime.UtcNow - new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc))
```

Comment:
Is there any way we can just not have to subtract Jan 1 1970?

Reply:
https://dirask.com/posts/C-NET-get-current-time-in-milliseconds-gp5BRj